composer.json

```
@@ -14,7 +14,7 @@
     "ext-json": "*",
     "ext-mbstring": "*",
     "ext-openssl": "*",
-    "monolog/monolog": "^1.16 || ^2.0"
+    "monolog/monolog": "^3.3"
```

Why dropping monolog 1 and 2 ?
And not simply allowing monolog 3 as following ?

```suggestion
    "monolog/monolog": "^1.16 || ^2.0 || ^3.0"
```

---

@pyrou 
You are right 👍🏻